x-pack/test/alerting_api_integration/common/fixtures/plugins/alerts/server/alert_types.ts

```diff
@@ -0,0 +1,483 @@
+/*
```

Comment:
File name is `alert_types.ts` but contains action types as well. Might be worth splitting into two separate files or one per type.

Reply:
👍